kolibri/plugins/coach/assets/src/app.js

```diff
@@ -196,10 +196,10 @@ class CoachToolsModule extends KolibriModule {
         },
         {
           name: PageNames.EXERCISE_RENDER,
-          path: '/:userId/:contentId/exercise-render',
+          path: '/:classId/exercise-render/:userId/:channelId/item/:contentId',
```

Comment:
Ah, are they still going to be sub-urls of those pages though? @indirectlylit would be nice to use an embeddable URL under each report, or link to a common URL from the report, no?